contracts/Cloudwalk/ErrorReporter.sol

```diff
@@ -133,7 +133,6 @@ contract TokenErrorReporter {
     error SetTrustedSupplierOwnerCheck();
     error SetTrustedBorrowerOwnerCheck();
 
-    error TrustedAdminAccountCheck();
     error TrustedBorrowerAccountCheck();
     error TrustedSupplierAccountCheck();
```

Comment:
Looks like the `TrustedSupplierAccountCheck` error is not used anywhere. Do we need it?

Reply:
Actually, this error has been removed, no?

Reply:
No :)
It is here.
The `TrustedAdminAccountCheck` error was removed.